web/message_bar.css

```diff
@@ -124,3 +124,97 @@
     }
   }
 }
+
+#editorUndoBar {
+  --text-primary-color: #15141a;
+
+  --message-bar-icon: url(images/secondaryToolbarButton-documentProperties.svg);
```

Comment:
Given where the current CSS file is being bundled, in order to prevent breakage of the `gulp components` build this image needs to be added to the following list.
https://github.com/mozilla/pdf.js/blob/9bf9bbda0b828f528679a981eb416b805880460e/gulpfile.mjs#L1147-L1155

Reply:
Great catch, I had no clue. Just fixed.